plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -240,4 +253,23 @@ private IcebergFileWriter createOrcWriter(
             throw new TrinoException(ICEBERG_WRITER_OPEN_ERROR, "Error creating ORC file", e);
         }
     }
+
+    public static OrcWriterOptions getOrcWriterOptions(Map<String, String> storageProperties, OrcWriterOptions orcWriterOptions)
```

Comment:
Is it possible to reuse the implementation we have in `Hive` ?

Reply:
Unfortunately, not yet. Iceberg implements map < string, string >.